Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -1010,6 +1010,6 @@ private static int GetSuspendSignal ()
         }
     }
 
-    [DllImport ("libc")] private extern static int killpg (int pgrp, int pid);
-    [DllImport ("libc")] private extern static int uname (nint buf);
+    [DllImport ("libc")] private static extern int killpg (int pgrp, int pid);
```

Comment:
Hm.

These attributes shouldn't be on the same line.

But now I just remembered you said you did this with just default configuration, yeah? That'll result in a few non-standard things like this.